synapse/rest/admin/users.py

```diff
@@ -683,3 +684,29 @@ async def on_PUT(self, request, user_id):
         await self.store.set_server_admin(target_user, set_admin_to)
 
         return 200, {}
+
+
+class UserMembershipRestServlet(RestServlet):
+    """
+    Get room list of an user.
+    """
+
+    PATTERNS = admin_patterns("/users/(?P<user_id>[^/]+)/rooms$")
```

Comment:
Can this be `/joined_rooms` to match the CS API one. I also prefer it as it more clearly states what its doing